azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -46,12 +49,16 @@
 
   public static final String COMMAND = "command";
   public static final String AZKABAN_MEMORY_CHECK = "azkaban.memory.check";
-  public static final String NATIVE_LIB_FOLDER = "azkaban.native.lib";
```

Comment:
Removing this parameter may break downstream plugins. I suggest we avoid this refactor. Just mark this deprecated for now.